modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -100,6 +100,65 @@ For example, type `cd /home/_your-user-name_/.bashrc`) in a text editor.
 . Save the `.bashrc` file.
 . Restart your terminal.
 
+== Preparing to Install Instio
+
+When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.
+
+=== Installing Instio 1.4x
+
+To install and configure Instio 1.4x for Anypoint Service Mesh:
+
+. Download and install Instio using the Instio documentation.
+. Enable the policy control flag:
++
+`set values.global.disablePolicyChecks=false`
++
+For more information about this flag, see https://archive.istio.io/v1.4/docs/tasks/policy-enforcement/enabling-policy/[Policy Enforcement]. 
```

Comment:
Would your audience really need to look this up to perform this task? 
I think this is probably unnecessary.

Reply:
I'd leave it in for people who need more clarity about what the flag does. Done! Thanks!